app-cli/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "captainduckduck",
-  "version": "1.0.13",
+  "version": "1.0.14",
```

Comment:
Typically, contributors don't up the version. The version gets changed just before publishing. It's okay for this PR, but typically this is not how it's done.